karma.conf.js

```diff
@@ -213,9 +189,9 @@ module.exports = function(config) {
       },
       module: {
         rules: rules
-      }
+      },
```

Comment:
@dompuiu are you ok with this? webpack5 isn't compiling to es5 by default for tests anymore, but I think we're getting everything down to `es5` when a library is built.